serving/src/main/resources/application.conf

```diff
@@ -73,6 +73,14 @@ defaults {
     ssl-verify = false
     ssl-verify = ${?SSL_VERIFY_CONFIG}
     # SSL Cert Configs for Git
+
+    #S3 settings
+    s3-download-timeout = 300 seconds
```

Comment:
Does it make sense to make this setting generic for all downloads? So instead of calling it the "s3" download timeout it would simply be, the download timeout?